src/airtable/rounds/funding_rounds.js

```diff
@@ -153,8 +158,8 @@ const calculateWinningProposalsForEarmark = (
       p.fields['USD Granted'] = oceanGranted * oceanPrice
       p.fields['OCEAN Granted'] = oceanGranted
       p.fields['Proposal State'] = 'Granted'
-      fundsLeft =
-        fundsLeft - usdGranted <= 0 ? fundsLeft : fundsLeft - usdGranted
+      fundsLeft = fundsLeft - usdGranted <= 0 ? 0 : fundsLeft - usdGranted
```

Comment:
Why not use `fundsLeft - usdGranted <= 0 ? fundsLeft`?

Reply:
If we can't fully grant we will give all the remaining funds which will lead to `fundsLeft = 0` , before we had `fundsLeft = fundsLeft` which means that we give the funds but the funds will still remain inside funds left.

Reply:
That makes perfect sense!